core/src/test/java/com/onelogin/saml2/test/settings/SettingBuilderTest.java

```diff
@@ -585,6 +589,128 @@ public void testFromProperties() throws IOException, Error, CertificateException
 	
 	/**
 	 * Tests SettingsBuilder constructor
+	 * Case: settings from values
+	 *
+	 * @see com.onelogin.saml2.settings.SettingsBuilder
+	 */
+	@Test
+	public void testLoadFromValues() throws Exception {
+	    	Map<String, String> prop = new LinkedHashMap<>();
```

Comment:
can you rename prop by samlData? 
```
Map<String, String> samlData = new LinkedHashMap<>();
```  
In the code we use prop as var of properties

```
private Properties prop = new Properties();
```


Also.. what do you think if we use
Map<String, Object> prop

and we check if the value is an boolean and convert it to its "string" version.. so we can accept:
samlData.put(SECURITY_NAMEID_ENCRYPTED, true);